Do not add -I$(top_srcdir) to x11vnc_CFLAGS, because we want to 
use <rfb/*> from libvncserver.

--- origsrc/x11vnc-0.9.9/x11vnc/unixpw.c	2010-02-12 01:59:25.010093800 -0600
+++ src/x11vnc-0.9.9/x11vnc/unixpw.c	2010-02-12 01:53:38.000000000 -0600
@@ -53,7 +53,7 @@ extern char *crypt(const char*, const ch
 #include "sslhelper.h"
 #include "cursor.h"
 #include "rates.h"
-#include <rfb/default8x16.h>
+#include <../rfb/default8x16.h>
 
 #if LIBVNCSERVER_HAVE_FORK
 #if LIBVNCSERVER_HAVE_SYS_WAIT_H && LIBVNCSERVER_HAVE_WAITPID
--- origsrc/x11vnc-0.9.9/x11vnc/userinput.c	2010-02-12 01:59:25.022094400 -0600
+++ src/x11vnc-0.9.9/x11vnc/userinput.c	2010-02-12 01:53:38.000000000 -0600
@@ -7755,7 +7755,7 @@ Bool xcheckmaskevent(Display *d, long ma
 #endif
 }
 
-#include <rfb/default8x16.h>
+#include <../rfb/default8x16.h>
 
 #define EVMAX 2048
 XEvent Ev[EVMAX];
